schema.sql

```diff
@@ -8,4 +8,28 @@ id INT Generated always as Identity,
 	neutered BOOLEAN,
 	weight_kg Decimal,
 	primary KEY (id)
-);
\ No newline at end of file
+);
+ALTER TABLE animals
+ADD species VARCHAR(50);
+BEGIN TRANSACTION;
+UPDATE animals
+SET species= 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] OPTIONAL:  To verify that change was made you can add `SELECT species from animals;` after SET species